test/mocks/upstream/cluster_info.h

```diff
@@ -28,7 +28,7 @@ class MockClusterInfo : public ClusterInfo {
   MOCK_CONST_METHOD0(connectTimeout, std::chrono::milliseconds());
   MOCK_CONST_METHOD0(perConnectionBufferLimitBytes, uint32_t());
   MOCK_CONST_METHOD0(features, uint64_t());
-  MOCK_CONST_METHOD0(httpCodecOptions, uint64_t());
+  MOCK_CONST_METHOD0(http2Settings, const Http::Http2Settings&());
```

Comment:
The tests are most likely failing because you need to setup a default ON_CALL return  a default Http2Settings object. You can verify by running the tests locally with ```--test_arg="--gmock_verbose=info"```